frontend/src/metabase/components/BrowseApp.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import { Box, Subhead } from "rebass";
+import { Link } from "react-router";
+
+import Question from "metabase-lib/lib/Question";
+
+import { MetabaseApi } from "metabase/services";
+
+export class DatabaseListLoader extends React.Component {
```

Comment:
Some comments on how we should replace these with `EntityListLoader`s might be good.